firefox-ios/Client/Frontend/PasswordManagement/PasswordManagerViewModel.swift

```diff
@@ -75,22 +75,21 @@ final class PasswordManagerViewModel {
     /// Searches SQLite database for logins that match query.
     /// Wraps the SQLiteLogins method to allow us to cancel it from our end.
     func queryLogins(_ query: String, completion: @escaping (([LoginRecord]) -> Void)) {
-        profile.logins.searchLoginsWithQuery(query).upon { result in
+        profile.logins.searchLoginsWithQuery(query, completionHandler: { result in
             ensureMainThread {
-                // Check any failure, Ex. database is closed
-                guard result.failureValue == nil else {
+                switch result {
+                case .success(let logins):
```

Comment:
Could you fix the warnings introduced here and on line 86 please? 🙏

Reply:
Can you please give me the warnings that you have? I don't have anything on these lines. Thanks 🙏

Reply:
Hi @dataports, I wanted to let you know that I've addressed the SwiftLint warning and pushed the changes. Thanks again 🙏

Reply:
@bmihai23 Just checking in, @PARAIPAN9 (thank you Sorin!) left a comment on this PR explaining potential ways to fix the failing test. You can try getting the Result block values in an array of `EncryptedLogin` if that makes sense for what you're trying to do here. Let me know if you need any additional help and I can try and help over on Element for quicker troubleshooting. Thanks for your patience!

Reply:
Hi @dataports, I just wanted to let you know that I solved these problems thanks to the helpful comment from @PARAIPAN9. Please let me know if everything is okay now. Thanks for helping me 🙏